test/integration/targets/s3_bucket/tasks/main.yml

```diff
@@ -6,10 +6,10 @@
     - name: set connection information for all tasks
       set_fact:
         aws_connection_info: &aws_connection_info
-          aws_access_key: "{{ aws_access_key }}"
-          aws_secret_key: "{{ aws_secret_key }}"
-          security_token: "{{ security_token }}"
-          region: "{{ aws_region }}"
+          aws_access_key: "{{ aws_access_key | default('') }}"
+          aws_secret_key: "{{ aws_secret_key | default('') }}"
+          security_token: "{{ security_token | default('') }}"
```

Comment:
This is convenient.

Reply:
Yes, I needed it to run it outside of ansible-test with my own profile so I could debug it